modules/simple_forms_api/app/services/simple_forms_api/pdf_stamper.rb

```diff
@@ -207,6 +207,15 @@ def self.stamp(desired_stamps, stamped_template_path, append_to_stamp, text_only
       File.rename(current_file_path, stamped_template_path)
     end
 
+    def self.verified_stamp(stamped_template_path, *, multiple: false, **)
+      orig_size = File.size(stamped_template_path)
+      command = multiple ? 'multistamp' : 'stamp'
+      send(command, stamped_template_path, *, **)
+      stamped_size = File.size(stamped_template_path)
+
+      raise StandardError, 'PDF stamping failed.' unless stamped_size > orig_size
+    end
```

Comment:
This solution works as intended but I'm not quite happy with how it's implemented. It obfuscates the `stamp` and `multistamp` method parameter lists. I'm thinking of actually pulling this logic into the respective methods. Any thoughts are welcome.